warp-element/src/element.js

```diff
@@ -2,7 +2,7 @@ import { PodiumElement } from "@podium/element";
 import { styles } from "./global.js";
 
 export default class WarpElement extends PodiumElement {
-  static styles = [...styles];
+  static styles = [styles];
```

Comment:
this changes to a single style object instead of an array of arbitrary length since we need to control the contents of a single object.